src/components/ScannerExamples.js

```diff
@@ -21,20 +22,26 @@ export default function ScannerExamples({ examples: allExamples }) {
       {Object.entries(exampleGroups).map(([targetName, examples]) => {
         return (
           <Tabs key={targetName}>
-            <h3>{targetName}</h3>
-            <TabList>
-              {examples.map(({ fileName }) => (
-                <Tab key={fileName}>{fileName.split(".")[0]}</Tab>
-              ))}
-            </TabList>
+            <Collapsible
```

Comment:
Cool Idea to use this here 👍

But this doesn't really work with the styling on the code block, as they are intentionally styled to be wider then the text. (negative margin)

<img width="1002" alt="Screen Shot 2020-08-25 at 15 57 12" src="https://user-images.githubusercontent.com/13718901/91183345-a64d2c80-e6eb-11ea-9bc2-7a0508261ba9.png">

Reply:
Might be fixable by allowing the overflow to be shown, unless that would break the collapsible.

Reply:
Done The Component itself has a property for that:  `overflowWhenOpen`.